Source/WebKit/UIProcess/Cocoa/GroupActivities/WKGroupSession.swift

```diff
@@ -57,13 +69,17 @@ fileprivate extension WKGroupSessionState {
     }
 
     var fallbackURL: URL? {
```

Comment:
Total nit: but I find these look nicer as one line when they’re trivial like this.
```suggestion
    var fallbackURL: URL? { urlActivity.webpageURL }
```